router/core/router.go

```diff
@@ -583,6 +582,16 @@ func (r *Router) newServer(ctx context.Context, routerConfig *nodev1.RouterConfi
 			subChiRouter.Use(metricHandler.Handler)
 		}
 
+		var subgraphs []Subgraph
```

Comment:
Pre-allocate the size based on the fixed numbers of subgraphs.

Reply:
Done